ember-cli-build.js

```diff
@@ -7,8 +7,6 @@ module.exports = function(defaults) {
     sassOptions: {
       includePaths: [
         "addon/styles",
-        "node_modules/frontend-basis/sass",
-        "node_modules/frontend-basis/node_modules"

```

Comment:
I'm sorry about that. I was using an old npm version (`v4`) when develop `ember-dialog`. For now it should work fine - I remade this config part for working with `npm@v6`.